core-sdk-samples/higgs-shop-sample-app/src/constants/index.ts

```diff
@@ -11,3 +11,6 @@ export type OrderPhaseTypes =
     | ORDER_PHASES.IN_PROGRESS
     | ORDER_PHASES.REVIEW
     | ORDER_PHASES.COMPLETE;
+
+export const APIkeyModalMessage =
+    "You have started the app without an API key in the config. There is information this app's README.md about how to add your API key.";
```

Comment:
I noticed that you need to restart the server so I figured we should just be explicit:

```suggestion
    "You have started the app without an API key in the config. For information on how to add your API key, see the README.md. Once you add your API key, restart your server for changes to take effect.";
```